app/models/admin.rb

```diff
@@ -0,0 +1,2 @@
+class Admin < ApplicationRecord
```

Comment:
Could you explain what this model will be used for?

Reply:
This is obsolete.  I was messing around before following a tutorial and we decided to go with User as the model.
I'm not exactly sure how to get rid of this though.  Any suggestions?

Reply:
Is it as simple as git rm on the file?

Reply:
Yup exactly! You can call that command for any files you want to remove from this PR.